pkg/addons2/addons.go

```diff
@@ -44,6 +45,12 @@ func InstallAddons(ctx context.Context, opts InstallOptions) error {
 		},
 	}
 
+	if opts.License != nil && opts.License.Spec.IsDisasterRecoverySupported {
```

Comment:
I still want to make this simpler. This function should not depend on that specific field and implementation of the license spec struct.

Have the caller (install2.go) make the determination if velero needs to be installed, and pass it in as a bool (IsVeleroInstalled, something like that). Right now, this jkust means that you'll pass that from opts.License.Spec into that bool field, but we remove the knowledge of license in this function.

The end result is that we have something that's more integration testable since we don't need to provide a license, we can provide the inputs to the function, the direction of what to do.